Packs/AWS/Integrations/AWS/AWS.py

```diff
@@ -1,27 +1,58 @@
 import demistomock as demisto  # noqa: F401
-from AWSApiModule import *  # noqa: E402
+from COOCApiModule import *  # noqa: E402
 from CommonServerPython import *  # noqa: F401
 from http import HTTPStatus
 from datetime import date
+from collections.abc import Callable
+from botocore.client import BaseClient as BotoClient
+from botocore.config import Config
 
+from boto3 import Session
 
-def test_module(params, command_args) -> str:
-    if test_account_id := params.get("test_account_id"):
-        command_args["account_id"] = test_account_id
-    else:
-        return "Please provide Test AWS Account ID for the Integration instance to run test"
+DEFAULT_MAX_RETRIES: int = 5
+DEFAULT_SESSION_NAME = "cortex-session"
+DEFAULT_PROXYDOME_CERTFICATE_PATH = "/etc/certs/egress.crt"
+DEFAULT_PROXYDOME = "10.181.0.100:11117"
```

Comment:
don't forget to replace with the env vars: `CRTX_HTTP_PROXY` and `EGRESSPROXY_CA_PATH`